### DISCUSSION TITLE
Making plugins to use package from system.

### DISCUSSION THREAD

**Discussion by semi710** [nix-community/nixvim#1969] [Q&A]
```nix
  plugins = {
    ltex-extra = {
      enable = true;
      # This has been made null so that nvix uses system latex package and nix run becomes faster (no downloads for test or build)
      # vimplugin-ltex_extra.nvim
      package = null;
    };
    vimtex.enable = true;
  };
```
  
  I am trying this to make ltex-extra use the ltex-extra package that is available in my system.

but this throws an error  

```
error: A definition for option `plugins.ltex-extra.package' is not of type `package'. Definition values:
       - In `/nix/store/fdn1lw1j5wlmpqxixr92hy8nvx7igshf-lang/tex.nix': null
```

**Comment by MattSturgeon** [comment]
I've opened an issue (#1970) from this to discuss whether this is something we can fix on our end, but my understanding is that `plugins.*.package` options need to be non-null because the package needs to be installed to neovim's runtimepath (not the system PATH env var).
